packages/core/src/tools/shell.ts

```diff
@@ -43,8 +43,12 @@ import {
 } from '../utils/shell-utils.js';
 import { SHELL_TOOL_NAME } from './tool-names.js';
 import type { MessageBus } from '../confirmation-bus/message-bus.js';
+import { DEFAULT_GEMINI_FLASH_MODEL } from '../config/models.js';
+import type { Content } from '@google/genai';
 
 export const OUTPUT_UPDATE_INTERVAL_MS = 1000;
+const SHELL_TIMEOUT_MS = 60000; // 1 minute initial timeout
```

Comment:
Timeout is probably the wrong verbiage here. Do you mean to say `TIMEOUT_ANALYSIS_CHECK_MS=60000`?